config/checklists/test-template.yaml

```diff
@@ -0,0 +1,14 @@
+name: "Monthly Meeting: %s"
+desc: Checklist for the execution of a Monthly Meeting
+lists: default
+cards:
+  - name: Update Meetup
+    desc: Adjust hoster and speaker information on meetup.com
+    due: -4 weeks
+  - name: Announce Meetup
+    desc: Click the announce button
+    due: -4 weeks
+  - name: Prepare Slides
+    desc: Click the announce button
```

Comment:
This should be `Create PR with slides for next meeting` or something

Reply:
@rdohms I also saw now that this hasn't been updated with the latest state of `monthly-meeting.yaml`. Did you miss it or is this file not being used anymore?